kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -449,10 +539,23 @@ if [ "$NETWORK_POLICY" != "" ]; then
     log_level -i "Setting network policy property."
     cat $AZURESTACK_CONFIGURATION | jq --arg NETWORK_POLICY $NETWORK_POLICY '.properties.orchestratorProfile.kubernetesConfig.networkPolicy=$NETWORK_POLICY' \
     > $AZURESTACK_CONFIGURATION_TEMP
-
+    
     log_level -i "Done setting network policy property."
 fi
 
+#####################################################################################
```

Comment:
I prefer not to deploy tiller, sorry I did not say that before

Reply:
so we are removing tiller entirely ??

Reply:
I vote yes, but we do not have to do it now

Reply:
Okay then.